secrets-provider-test-chart/values.yaml

```diff
@@ -0,0 +1,25 @@
+global:
+  namespace: local-secrets-provider
+
+# K8s permission defaults
+rbac:
+  roleName: secrets-provider-role
+  roleBinding: secrets-provider-role-binding
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: true
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
```

Comment:
I don't understand this comment.
Also I think this is incorrect behavior, we shouldn't generate this name.
We should either use a default, or get one from `custom-values.yaml`.

Reply:
@sigalsax - are you sure that "create" is valid attribute here ?